tools/processresults.py

```diff
@@ -234,7 +234,12 @@ def _CanonicalizeQuestions(results):
   if not fixed_results:
     raise UnexpectedFormatException('No results with questions found')
 
-  _ReorderAttributeQuestions(fixed_results, ATTRIBUTE_QUESTION_PREFIX)
+  try:
+    _ReorderAttributeQuestions(fixed_results, ATTRIBUTE_QUESTION_PREFIX)
+  except UnexpectedFormatException as e:
+    # Print UnexpectedFormatException and continue, since this is usually
+    # due to lack of attribute questions in survey, which is fine.
+    print 'Exception from _ReorderAttributeQuestions: %s' % (e.value)
```

Comment:
You might want to consider the logging API in the future, if printing becomes more common: https://docs.python.org/2/library/logging.html